### PR TITLE
Label for username and password changed

### PR DESCRIPTION
**Pull request by rachelrajanexp**
# Changelog
- ###  Index Page

   - Label for the Username field has been changed

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -51,12 +51,12 @@ <h2 class="login-here" >
                     />
                   </div>
                   <div class="form-group last mb-4">
-                    <label for="username">Username & Password</label>
+                    <label for="username">Username and Password</label>
```
Comment (poojaviswam) [inline html/index.html:54]: Please rename to User name *and* Password

File: html/login.html [hunk 2]
```diff
@@ -3,13 +3,13 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Firebase Auth Navbar Page</title>
+    <title>Firebase Auth Navbar Yes</title>
```
Comment (poojaviswam) [inline html/login.html:6]: Replace Yes with No

**Review by poojaviswam — CHANGES_REQUESTED** [review]

File: html/login.html (hunk 2 above)
Comment (rachelrajanexp) [reply html/login.html]: Resolved

Comment (rachelrajanexp) [reply html/login.html]: Resolved

**Review by poojaviswam — APPROVED** [review]

**Pull request merged**